style.css

```diff
@@ -4202,13 +4237,16 @@ ul {
 }
 
 .search-result-breadcrumbs {
+  display: table-row;
```

Comment:
Could we use something other than `table-row`, e.g. `inline-block`, since we're not within a table?

Reply:
there might be a better way here. I used `table-row` and `table-cell` to make sure `.search-result-breadcrumbs li` has the same height as `.search-result-breadcrumbs` such that they are aligned vertically with `meta-data`

Reply:
I think we used those at the time to keep support for older browsers that we no longer support. But you have a point, let's keep it consistent 👍